### DISCUSSION TITLE
Genymotion Desktop touchpad and keyboard input not working on m1.

### DISCUSSION THREAD

**Discussion by miknonny** [Genymobile/feedback#60] [Genymotion Desktop] [answered]
Hi,

Installed genymotion desktop on mac m1. all went smoothly. but i do not seem to be able to use my touchpad and keyboard. could i be missing something because I have used previously on my windows device.

thanks.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @miknonny,
This board is for feature requests only ; it is not suitable for technical assistance.
Please contact our technical support at https://support.genymotion.com/hc/en-us/requests/new
Thank you.
